syntaxes/src/build.ts

```diff
@@ -5,11 +5,12 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-
 import * as fs from 'fs';
 
-import {GrammarDefinition, JsonObject} from './types';
-import {template} from './template/grammar';
+import { template } from './template/grammar';
+import { expression } from './expression/grammar';
+import { GrammarDefinition, JsonObject } from './types';
```

Comment:
I think you will have to run the formatter (`yarn format`); the CI style check will probably fail on this file.